tools/NuGetProj.targets

```diff
@@ -92,6 +92,11 @@ Common build targets overwrites.
     <CleanDependsOn>BeforeClean;ReadNuGetCleanOutputs;CoreClean;AfterClean</CleanDependsOn>
     <RebuildDependsOn>Clean;Build</RebuildDependsOn>
   </PropertyGroup>
+  <!-- FIXME disable nuget pack on Unix as it currently fails -->
```

Comment:
Rather than make this change in this tools file, can you make it in the WebJobs.Script.proj to exclude the nuget packaging?